dev/how-to/deploy/tidb-operator.md

```diff
@@ -13,11 +13,11 @@ category: how-to
 TiDB Operator 部署前，请确认以下软件需求：
 
 * Kubernetes v1.10 或者更高版本
+* 如果要使用 zone-aware 持久化卷, 则需要 Kubernetes v1.12 或者更高版本
```

Comment:
All our tests are based on 1.12, so I think it's better to require the minimal cluster version to 1.12.